qiskit/backends/ibmq/ibmqbackend.py

```diff
@@ -46,33 +47,46 @@ def __init__(self, configuration, api=None):
         """
         super().__init__(configuration=configuration)
         self._api = api
-
+        self._submit_info = None   # dictionary coming from api upon submission
```

Comment:
Good point. The primary reason for this was to be able to obtain a job_id for IBM Q jobs so it could be passed to the IBMQJob instance. Let me see if I can avoid this.